var/spack/repos/builtin/packages/numaprof/package.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Numaprof(CMakePackage):
+    """
+    NumaProf is a NUMA memory access profiling tool. It is based on Intel-PIN to intercept
+    all the memory access and report them on NUMA counters so we can tell by annotating the
+    source code where you make local, remote, unpinned memory accessed. It also provide
+    some charts to better understand the NUMA behavior of the application.
+    """
+
+    # Infos
+    homepage = "https://memtt.github.io/numaprof"
+    url = "https://github.com/memtt/numaprof/releases/download/v1.1.4/numaprof-1.1.4.tar.bz2"
+    maintainers = ["svalat"]
```

Comment:
The maintainers syntax has change to the following:  

```suggestion
    maintainers("svalat")
```

Reply:
Done